### ISSUE TITLE
Tax applied to other countries when it shouldn't be

### ISSUE THREAD

**Issue by amdrew** [easydigitaldownloads/Easy-Digital-Downloads#2088]
EDD 1.9.6

It appears that tax is applied to all countries when it shouldn't be, except for 3 countries which show the correct behaviour of no tax being applied:

USA, CAN, AUS

For example, if tax is setup as per the screenshot below, only NZ should exhibit a tax rate of 15%. If you change countries, all countries except the above 3 add the tax on. This should not be the case.

Here are the tax settings
https://cloudup.com/cn7Ze7ZwYZS

Here are my findings from testing
https://easydigitaldownloads.com/support/topic/split-tax-charged-when-it-shouldnt-be/page/2/#post-196643

To summarize:

Testing the customer's demo:

Logged in – no issues at all
Logged out – I can replicate the issues

On my local install

Logged in – I can replicate the issues
Logged out – I can replicate the issues
